src/BSON/Binary.stub.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+  * @generate-class-entries static
+  * @generate-function-entries static
+  */
+
+namespace MongoDB\BSON
+{
+    final class Binary implements BinaryInterface, JsonSerializable, Type, \Serializable
+    {
+        /** @var int */
+        public const TYPE_GENERIC = 0x01;
```

Comment:
I've added these in a separate commit since we used to use the libbson enum to define these constants. Since these are protected by the BC promise, I don't mind hardcoding the values here but can revert the change if you'd rather use libbson values.

Reply:
Ah, I didn't consider that. Since `php_phongo_binary_init_ce` isn't going away, I think keeping the `zend_declare_class_constant_long` lines in there is sensible. This is going to come up with other files as well, and I'd rather depend on the libmongoc constants if possible, since that might also catch a bug on their end.

If not, I'd at least want to have comments in these stub files referring to the libmongoc constant we're hard-coding. For example:

```
public const TYPE_GENERIC = 0x01; /* BSON_SUBTYPE_BINARY */
```

Reply:
Try (may requires 8.2 to generate the arginfo)
```
/** @cvalue BSON_SUBTYPE_BINARY */
public const TYPE_GENERIC = UNKNOWN;
```

See https://github.com/php/php-src/commit/1362fefc85ea8da7e32338291fa36182745c9622

Reply:
Ah, that is nice. I was already using the 8.2 file (the previous version didn't handle constants), so I switched to using `@cvalue`.